tests/Aspire.Hosting.Testing.Tests/TestingBuilderTests.cs

```diff
@@ -14,8 +14,8 @@ public class TestingBuilderTests
     public async Task HasEndPoints()
     {
         var appHost = await DistributedApplicationTestingBuilder.CreateAsync<Projects.TestingAppHost1_AppHost>();
+        appHost.Configuration[KnownConfigNames.AllowUnsecuredTransport] = "true";
```

Comment:
`DistributedApplicationTestingBuilder` turns off the dashboard, so we don't need this.